docker-compose.yaml

```diff
@@ -9,3 +9,11 @@ services:
     platform: linux/amd64
     environment:
       - DEPOT_TOOLS_UPDATE=0
+  unittest:
+    build:
+      context: cobalt/docker/unittest
+      cache_from:
+        - ghcr.io/youtube/cobalt/unittest:latest
+    image: ghcr.io/youtube/cobalt/unittest:latest
+    environment:
+      - DEPOT_TOOLS_UPDATE=0
```

Comment:
You likely don't need this, unless you are also running `gclient` in this container

Reply:
removed